courses/machine_learning/deepdive2/structured/solutions/3c_bqml_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
<ul><li>Since this is the solution notebook, the wording should be more a description of what we are doing than an exercise statement. I'd replace this cell with something similar to the following:</li></ul><p> </p><p>## DNN Regressor Model with BigQuery</p><p> </p><p>Let's train a DNN regressor model in BQ using <code>MODEL_TYPE=DNN_REGRESSOR</code> with 2 hidden layers with 64 and 32 neurons each (<code>HIDDEN_UNITS=[64, 32]</code>) and a batch size of 32 (<code>BATCH_SIZE=32</code>):</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done</p>